Makefile

```diff
@@ -113,12 +113,22 @@ endif
 	find . -name \*~ -delete
 	find . -name \#\* -delete
 	rm -f bin/crio
+	rm -f bin/crio.cross.*
```

Comment:
Why are we picking on subsets of `bin`?  Can't we just use a single `rm -rf bin`?